stdlib/public/core/HackyTensorflowMigrationSupport.swift

```diff
@@ -0,0 +1,77 @@
+//===-- TensorGroup.swift -------------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the TensorGroup type.
+//
+//===----------------------------------------------------------------------===//
+
+// This whole file is a hack in order to allow moving TensorFlow out of the
+// swift compiler and to build it independently. There is an obviously more
+// general version of this using associated types or something.
```

Comment:
Is this file going to stay in the stdlib in the long run? 

If yes, I suggest to call this file `TensorFlow.swift` and change this comment to something like "This file defines a minimal surface of TensorFlow-related data structures that is necessary for Swift for TensorFlow compiler". Rationale: if this design is here to stay, let's own that instead of deferring to indefinite future improvements.

Reply:
No. It should go away. Naming things in such a way may encourage it to stay around longer than it needs to.